scripts/EN/Harry

```diff
@@ -3335,7 +3335,7 @@ CleanScreen(); // 0x4072: 00000064
 LoadBackground(0x54); // 0x4076: 00001069
 DrawBackground(); // 0x407c: 00000065
 PlayMusic(0x6e); // 0x4080: 0000106c
-ChoiceIdx(0x5, (160)"If I use the pet bottle...", (161)"A ) Scoop up the red liquid.", (162)"B ) Scoop up the blue liquid.", (163)"C ) Scoop up the mixture of purple liquid."); // 0x4086: 00005035
+ChoiceIdx(0x5, (160)"If I use the plastic bottle…", (161)"A) Scoop up the red liquid.", (162)"B) Scoop up the blue liquid.", (163)"C) Scoop up the mixture of purple liquid."); // 0x4086: 00005035
```

Comment:
Changed pet to plastic to match the PS1 version. (PET refers to the type of plastic and is how they refer to these bottles in Japan.)